CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Unreleased
 
 # v1.6.0
+## Enhancements
 * Add `Name` field to `OAuthClient` by @barrettclark [#466](https://github.com/hashicorp/go-tfe/pull/466)
+* Add support for creating both public and private `RegistryModule` without VCS by @Uk1288 [#460](https://github.com/hashicorp/go-tfe/pull/460)
```

Comment:
Since it was previously possible to create private registry modules, I think it would be suffice to say that we added the ability to create public registry modules

```suggestion
* Add support for creating public `RegistryModule` (with no VCS connection) by @Uk1288 [#460](https://github.com/hashicorp/go-tfe/pull/460)
```